src/maidsafe/routing/tests/close_nodes_change_test.cc

```diff
@@ -385,6 +385,37 @@ TEST_F(CloseNodesChangeTest, BEH_FullSizeRoutingTable) {
   }
 }
 
+TEST_F(CloseNodesChangeTest, BEH_Constructors) {
```

Comment:
This should be a type-parameterised test rather than repeating the identical contents of the test three times.